bitnami/prestashop/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: prestashop
-version: 11.0.0
+version: 11.0.1
```

Comment:
As far as I know, changing the secret key will be a breaking change, which would mean a new major. I'm not really sure about the necessity of this change to justify a new major.